packages/sync-actions/src/categories.js

```diff
@@ -3,6 +3,7 @@ import type { SyncAction, ActionGroup } from 'types/sdk'
 import flatten from 'lodash.flatten'
 import createBuildActions from './utils/create-build-actions'
 import createMapActionGroup from './utils/create-map-action-group'
+import actionsMapCustom from './utils/create-map-action-custom'
 import * as categoryActions from './category-actions'
```

Comment:
Is this preferred over importing each object as a separate variable?

Reply:
It depends a bit. In this case it a. seems to be a pattern in the code base and b. often times when destructuring the named exports you lose a bit of context where they are from.

Reply:
Thanks.

(b) is the explanation I've seen often in articles.